### ISSUE TITLE
6.2 versions fail on build with typescript / missing semver in type declarations.

### ISSUE THREAD

**Issue by mwawrusch** [magiclabs/magic-js#235]
### 🐛 Description

I have a problem when trying to use the latest (and maybe earlier) version of the magic sdk for the browser. I get the following errors after running with 6.2 during compilation (typescript, angular). Previous version that I know it worked was 5.x:

```
Error: node_modules/@magic-sdk/provider/dist/types/util/index.d.ts:5:25 - error TS2307: Cannot find module './semver' or its corresponding type declarations.

5 export * as semver from './semver';
```

### 🧩 Steps to Reproduce

1. Create a project that uses typescript (and maybe angular). 
2. add magic 6.2.

### 🤔 Expected behavior

It should compile

### 😮 Actual behavior

```
Error: node_modules/@magic-sdk/provider/dist/types/util/index.d.ts:5:25 - error TS2307: Cannot find module './semver' or its corresponding type declarations.

5 export * as semver from './semver';
```

### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `magic-sdk`      | 6.x
| Browser          |  n/a
| `yarn`           | npm 6.14.11
| Operating System | Mac

**Comment by smithki** [comment]
Please help us isolate the issue with the following:

1. Your version of TypeScript
2. The content of your `tsconfig.json` file
3. A reproducible example

**Comment by smithki** [comment]
Apparently you can work around this in the meanwhile with `"skipLibCheck": true` in your `tsconfig.json`. We'll investigate the type errors once our engineers have cycles to do so.

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-ext/bitcoin@1.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/bitcoin%401.0.1), [`@magic-ext/oauth@0.11.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/oauth%400.11.1), [`@magic-ext/react-native-oauth@0.6.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-oauth%400.6.1), [`@magic-ext/webauthn@0.3.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/webauthn%400.3.1), [`@magic-sdk/commons@2.2.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/commons%402.2.1), [`@magic-sdk/pnp@0.5.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/pnp%400.5.1), [`@magic-sdk/provider@6.2.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%406.2.1), [`@magic-sdk/react-native@6.2.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native%406.2.1), [`magic-sdk@6.2.1`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%406.2.1) :rocket:
